src/chatbot/bot.py

```diff
@@ -75,6 +76,32 @@ async def show_score(ctx: Context) -> None:
     await ctx.send(f"Your social score is currently {user_to_lookup.social_score}")
 
 
+@bot.command()
+@is_in_channel(set(["bot-commands"]))
+async def register_score(
+    ctx: Context, emoji: Union[PartialEmoji, str], score: int
+) -> None:
+    if isinstance(emoji, PartialEmoji):
+        emoji_id = str(emoji.id)
+    else:
+        emoji_id = emoji
+    logger.info("Registering emoji_id %s with score %d", emoji_id, score)
+    emoji_score: EmojiScore = await EmojiScore.objects.async_get(emoji_id=emoji_id)
+    emoji_score.score = score
+    await emoji_score.async_save()
+    await ctx.message.reply("Your emoji score was registered/updated successfully")
+
+
+@register_score.error
+async def register_score_error(ctx: Context, error: CommandError) -> None:
+    if isinstance(error, CheckFailure):
+        logger.debug(
+            "register_score called in channel %s which is not allowed", ctx.channel
+        )
+    else:
+        logger.error("Error executing register_score", exc_info=error)
+
+
 @bot.event
 async def on_raw_reaction_add(payload: RawReactionActionEvent) -> None:
```

Comment:
Add and remove are basically exactly the same function now - can we deduplicate the code?